README.md

```diff
@@ -101,27 +97,20 @@ The Kin Ecosystem SDK Sample App demonstrates how to perform common workflows su
 
 >**NOTE:** The Sample App is for demonstration only, and should not be used for any other purpose.
 
-The Sample App is pre-configured with the default whitelist credentials ```appId='test'``` and 
-```apiKey='AyINT44OAKagkSav2vzMz'```. These credentials can be used for integration testing in any app, but authorization will fail if you attempt to use them in a production environment.
-
-You can also request unique apiKey and appId values from Kin, and override the default settings, working either in whitelist or JWT authentication mode.
+The Sample App is pre-configured with the default credentials ```appId='test'``` and
+```jwt private key```. These credentials can be used for integration testing in any app, but authorization will fail if you attempt to use them in a production environment.
 
 *To override the default credential settings:* 
 
 Create or edit a local ```credential.properties``` file in the ```app``` module directory and add the lines below, using the ```appId``` and ```apiKey``` values you received.
 
 ```
-APP_ID="YOUR_APP_ID" // For whitelist registration, and also as the issuer (iss). Default = 'test'.
+APP_ID="YOUR_APP_ID" // The issuer (iss). Default = 'test'.
 
-API_KEY="YOUR_API_KEY" // For whitelist registration. Default = 'AyINT44OAKagkSav2vzMz'.
+ES256_PRIVATE_KEY="YOUR_ES256_PRIVATE_KEY” // For production, JWT is created by server side with ES256 signature.
```

Comment:
The sample app support RS512
the key is RS512_PRIVATE_KEY (don't know why we write ES256_PRIVATE_KEY)

Reply:
👍